source/camera/CameraManager.h

```diff
@@ -62,7 +66,7 @@ class CameraManager
         fullStop
     };
 
-    CameraManager(Ogre::SceneManager* sceneManager, TileContainer* gameMap, Ogre::RenderWindow* renderWindow);
+    CameraManager(Ogre::SceneManager* sceneManager, GameMap* gameMap, Ogre::RenderWindow* renderWindow);
```

Comment:
Why did you change TileContainer to GameMap ? TileContainer is not a very useful abstract level since it is almost not used but why did you change ?